solo-chains/runtime/dancelight/src/bridge_to_ethereum_config.rs

```diff
@@ -17,11 +17,46 @@
 //! The bridge to ethereum config
 
 pub const SLOTS_PER_EPOCH: u32 = snowbridge_pallet_ethereum_client::config::SLOTS_PER_EPOCH as u32;
+
 use {
-    crate::{parameter_types, weights, Runtime, RuntimeEvent},
+    crate::{
+        parameter_types, weights, AggregateMessageOrigin, Balance, Balances, EthereumOutboundQueue,
+        EthereumSystem, FixedU128, GetAggregateMessageOrigin, Keccak256, MessageQueue, Runtime,
+        RuntimeEvent, TreasuryAccount, WeightToFee, UNITS,
+    },
+    pallet_xcm::EnsureXcm,
     snowbridge_beacon_primitives::{Fork, ForkVersions},
+    snowbridge_core::{gwei, meth, AllowSiblingsOnly, PricingParameters, Rewards},
+    sp_core::{ConstU32, ConstU8},
 };
 
+parameter_types! {
+    pub Parameters: PricingParameters<u128> = PricingParameters {
+        exchange_rate: FixedU128::from_rational(1, 400),
```

Comment:
I am curious about this exchange rate. Was this copied from the snowbridge config?

Reply:
*disregard*